src/main.ts

```diff
@@ -403,6 +403,7 @@ export async function createPipeline(config: Config): Promise<string> {
     const executionGraphId = locationHeader.substring(locationHeader.lastIndexOf("/") + 1)
     return executionGraphId
   } catch (error) {
+    core.debug(`Error: ${JSON.stringify(error)}`)
```

Comment:
Coming late to the party but I think you have this line duped.

Reply:
Thanks! This will be fixed in https://github.com/vmware-labs/vmware-image-builder-action/pull/78